### ISSUE TITLE
下载配置文件后再导入显示导入失败

### ISSUE THREAD

**Issue by giteeeeee** [Akegarasu/lora-scripts#337]
1.8.3版本，windows环境，brave/edge浏览器

如下图黄色箭头所示，先点击下载配置文件获得xxx.toml，然后点击导入配置文件，会显示导入失败

![147](https://github.com/Akegarasu/lora-scripts/assets/89521447/71248a38-5e16-4cbd-a795-90dcf5980fc4)

导入失败：

![-18 204351](https://github.com/Akegarasu/lora-scripts/assets/89521447/d3bc3158-6c87-4c99-9395-4786bdd3653e)

lora和dreambooth训练都是这样，无法导入配置文件

以下为其中一个toml文件的参数：
model_train_type = "sdxl-lora"
pretrained_model_name_or_path = "./sd-models/animagineXLV3_v30.safetensors"
v2 = false
train_data_dir = "E:/StableDiffusion/lora-scripts/train/aki/1_fx1"
prior_loss_weight = 1
resolution = "1024,1024"
enable_bucket = true
min_bucket_reso = 512
max_bucket_reso = 2_048
bucket_reso_steps = 32
output_name = "aki22223_1e3"
output_dir = "E:/StableDiffusion/"
save_model_as = "safetensors"
save_precision = "fp16"
save_every_n_epochs = 12_511
max_train_epochs = 125
train_batch_size = 1
gradient_checkpointing = false
network_train_unet_only = false
network_train_text_encoder_only = false
learning_rate = 0.0001
unet_lr = 0.0001
text_encoder_lr = 0.001
lr_scheduler = "constant"
lr_warmup_steps = 0
optimizer_type = "AdamW8bit"
network_module = "networks.lora"
network_dim = 32
network_alpha = 32
log_with = "tensorboard"
logging_dir = "./logs"
caption_extension = ".txt"
shuffle_caption = false
keep_tokens = 0
max_token_length = 255
seed = 1_337
clip_skip = 2
mixed_precision = "fp16"
xformers = true
lowram = false
cache_latents = true
cache_latents_to_disk = false
persistent_data_loader_workers = true
gpu_ids = [ "0" ]
